patches/chrome-android-BUILD.gn.patch

```diff
@@ -42,3 +42,11 @@ index b067a3f7af514bc48632365427ba8ecffa4885df..dadadab8dd95e47bc228053bd1335cbd
        if (_is_monochrome) {
          deps += [ "//android_webview:locale_pak_assets" ]
          if (webview_includes_weblayer && !_is_bundle_module) {
+@@ -2286,6 +2291,7 @@ android_library("base_module_java") {
+     "//components/payments/content/android:service_java",
+     "//third_party/androidx:androidx_browser_browser_java",
+   ]
++  sources += brave_java_base_module_sources deps += brave_java_base_module_deps
```

Comment:
seems like maybe we can consolidate some patches into this one since this always gets included?

Reply:
maybe as a follow-up?

Reply:
I don't see any other modifications to this part. And I think it is better not mixing other issues into this one.